contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -344,6 +345,7 @@ babelfish_helpdb(PG_FUNCTION_ARGS)
 	if (PG_NARGS() > 0)
 	{
 		dbname = TextDatumGetCString(PG_GETARG_DATUM(0));
+		dbname = str_tolower(dbname, strlen(dbname), DEFAULT_COLLATION_OID);
```

Comment:
if you overwrite this ptr, line 352's error message will show the lower case dbname that used for internal handling